com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -826,11 +831,11 @@ void ResetData()
         public virtual void Initialize() { }
 
         /// <summary>
-        /// Implement `Heuristic()` to choose an action for this agent using a custom heuristic.
+        /// Implement <see cref="IHeuristicProvider"/> to choose an action for this agent using a custom heuristic.
```

Comment:
I think this is OK referring to Agent.Heuristic since it's talking specifically about the Agent. Same below in the comments on OnActionReceived()